editor/src/components/inspector/inspector.tsx

```diff
@@ -349,6 +351,16 @@ export const Inspector = React.memo<InspectorProps>((props: InspectorProps) => {
     'RootElementIndicator aRootElementIsSelected',
   )
 
+  const areGroupChildren = useEditorState(
```

Comment:
```suggestion
  const areChildrenOfAGroup = useEditorState(
```
Small beans, but the name makes me think this is checking the children to see if they are groups.

Reply:
I went a little more with `allSelecteElementsdAreChildrenOfAGroup`